lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -99,6 +110,12 @@ internal sealed class IMRUDriver<TMapInput, TMapOutput, TResult> : IObserver<IDr
             _perMapperConfigs = perMapperConfigs;
             _allocatedUpdateTaskEvaluator = false;
             _completedTasks = new ConcurrentBag<ICompletedTask>();
+            _allowedFailedEvaluators = (int) (failedEvaluatorsFraction*dataSet.Count);
+            _currentFailedEvaluators = 0;
```

Comment:
You can move this assignment to the field definition. That way, we can't forget it when we ever add a second constructor.